riptide-core/src/main/java/org/zalando/riptide/Route.java

```diff
@@ -115,4 +118,39 @@ static Route call(final ThrowingConsumer<ClientHttpResponse, ? extends Exception
         ThrowingConsumer<T, Exception> andThen(final ThrowingConsumer<R, ? extends Exception> consumer);
     }
 
+    static Route named(final Object name, final Route route) {
+        return new Route() {
+            @Override
+            public void execute(final ClientHttpResponse response, final MessageReader reader) throws Exception {
+                route.execute(response, reader);
+            }
+
+            @Override
+            public String toString() {
+                final ThrowingSupplier<String, Exception> stringer = () -> {
```

Comment:
Yes, there is no way to provoke those exceptions and therefore you can' test it.